src/main/kotlin/com/openlattice/organization/DatasetApi.kt

```diff
@@ -0,0 +1,167 @@
+package com.openlattice.organization
+
+import retrofit2.http.*
+import java.util.*
+
+interface DatasetApi {
+
+    companion object {
+        /* These determine the service routing */
+        const val SERVICE = "/datastore"
+        const val CONTROLLER = "/organization-database"
```

Comment:
This is just a general comment, but we've been super inconsistent in these controller paths between different patterns of `first-second` vs `first_second` vs `firstsecond`vs `first/second` so we should probably collectively pick whichever one we want and stick to that pattern moving forward